tokenize_from_csv/README.md

```diff
@@ -1,58 +1,73 @@
---header "Authorization: Bearer ${DATABRICKS_TOKEN}" \
---data '{
-    "scope": ${DATABRICKS_SCOPE},
-    "key": "${CONFIG_KEY_NAME}",
-    "string_value": "{\"creds_path\":\"/Volumes/main/default/testvol/databricks-tokenize-demo-creds.json\",\"vault_url_prefix\":\"https://abcdefg.vault.skyflowapis.com\", \"vault_id\":\"${VAULT_ID_VALUE}\",\"account_id\":\"${ACCOUNT_ID_VALUE}\"}"
-}'
-```
+    ```json
+    {
+        "account_id": "a451b783713e4424a7d761bb7bbc84eb",
+        "vault_url_prefix": "https://abfc8bee4242.vault.skyflowapis.com",
+        "vault_id": "c35dcf755b60479cae67ae1362e49643",
+        "creds_path": "/Volumes/main/default/test_volume/credentials.json"
+    }
+    ```
 
-- `$DATABRICKS_HOST` is the URL for your databricks instance
-- `$DATABRICKS_TOKEN` is a bearer token which can be obtained through your account settings in the Databricks UI
-- `$DATABRICKS_SCOPE` is the scope for your databricks secrets (see https://docs.databricks.com/en/security/secrets/secret-scopes.html)
-- `$CONFIG_KEY_NAME` is the name for the config secret
+1.  Upload the stringified JSON object to a Databricks secret. You can do this using the Databricks REST API. For example:
 
-##### Using the UDF
+    ```bash
+    curl --request POST "https://${DATABRICKS_HOST}/api/2.0/secrets/put" \
+    --header "Authorization: Bearer ${DATABRICKS_TOKEN}" \
+    --data '{
+        "scope": ${DATABRICKS_SCOPE},
+        "key": "${CONFIG_KEY_NAME}",
+        "string_value": "{\"account_id\":\"$ACCOUNT_ID\",\"vault_url_prefix\":\"$VAULT_URL\",\"vault_id\":\"$VAULT_ID\",\"creds_path\":\"$CREDENTIALS_PATH\"}"
+    }'
+    ```
 
-1. Upload the target CSV file into a Databricks volume. Note the path to this file as `$INPUT_CSV`. You can obtain the full path of the file in Databricks by right-clicking on the file and selecting "copy file path".
-2. Upload the configuration JSON to Databricks secret scope as explained above.
-3. Copy this UDF into a Databricks notebook. Click the `run` button on top of the notebook. This registers (or re-registers) the UDF.
-4. Create a new cell. Select SQL as the language. (You can also skip this step if you plan to use the function outside of Databricks UI).
-5. Call the UDF. Ex: `SELECT tokenizeCSV("/Volumes/main/default/testvol/fake_data.csv", "table_for_csv",  MAP("name", "name", "email", "email")) AS detokenized_data;`
\ No newline at end of file
+## Use the UDF
+
+1. Create a map that maps the CSV columns to the Skyflow columns. For example, if `fake_data.csv` has the `customer_names`, `user_email_address`, and `social_security_number` columns, the map could be:
+
+    | CSV column | Skyflow column |
+    |------------|----------------|
+    | customer_names | name |
+    | user_email_address | email_address |
+    | social_security_number | ssn |
+
+1. Copy the contents of the `tokenize_from_csv.py` file into a Databricks notebook. Click the `run` button on top of the notebook. This registers (or re-registers) the UDF.
```

Comment:
1,2,3

Reply:
Keeping it as-is. Markdown autoenumerates the list items. Keeping it all `1`s keeps the source clean, and you don't need to worry about renumbering every time you update the number or order of steps.

Reply:
Super useful, thank you